Source/SwiftLintFramework/Rules/RuleConfigurations/DirectoryNameNoSpaceConfiguration.swift

```diff
@@ -0,0 +1,34 @@
+public struct DirectoryNameNoSpaceConfiguration: RuleConfiguration, Equatable {
+    public var consoleDescription: String {
+        return "(severity) \(severity.consoleDescription), " +
+            "excluded: \(excluded.sorted()), " +
+            "parentDirectory: \(parentDirectory)"
+    }
+
+    public private(set) var severity: SeverityConfiguration
+    public private(set) var excluded: Set<String>
+    public private(set) var parentDirectory: String
```

Comment:
I wonder if we should infer this from the working directory or the configuration directory instead

Reply:
I can't infer it from the [currentWorkingDirectory](https://developer.apple.com/documentation/foundation/filemanager/1410766-currentdirectorypath) because that is where the process is actually executing (DerivedData).

Reply:
I'm open to inferring from the configuration directory, but I'm not sure how to access that.